microprofile.jdt/org.eclipse.lsp4mp.jdt.test/src/main/java/org/eclipse/lsp4mp/jdt/core/BasePropertiesManagerTest.java

```diff
@@ -58,42 +58,33 @@
-		all_quarkus_extensions("all-quarkus-extensions"), //
-		config_hover("config-hover"), //
-		config_properties("config-properties"), //
-		config_quickstart("config-quickstart"), //
-		config_quickstart_test("config-quickstart-test"), //
-		empty_maven_project("empty-maven-project"), //
-		other_empty_maven_project("empty-maven-project"), //
-		folder_name_different_maven("folder-name-different-maven"), //
-		hibernate_orm_resteasy("hibernate-orm-resteasy"), //
-		hibernate_orm_resteasy_yaml("hibernate-orm-resteasy-yaml"), //
-		kubernetes("kubernetes"), //
-		microprofile_fault_tolerance("microprofile-fault-tolerance"), //
-		microprofile_health_quickstart("microprofile-health-quickstart"), //
-		microprofile_lra("microprofile-lra"), //
-		microprofile_context_propagation("microprofile-context-propagation"), //
-		microprofile_metrics("microprofile-metrics"), //
-		microprofile_opentracing("microprofile-opentracing"), //
-		microprofile_openapi("microprofile-openapi"), //
-		microprofile_reactive_messaging("microprofile-reactive-messaging"), //
-		microprofile_graphql("microprofile-graphql"), //
-		microprofile_jwt_quickstart("microprofile-jwt-quickstart"), //
-		quarkus_container_images("quarkus-container-images"), //
-		rest_client_quickstart("rest-client-quickstart"), //
-		using_vertx("using-vertx");
+	public static class MicroProfileMavenProjectName {
+
+		public static String all_quarkus_extensions="all-quarkus-extensions";
+		public static String config_hover="config-hover";
+		public static String config_properties="config-properties";
+		public static String config_quickstart="config-quickstart";
+		public static String config_quickstart_test="config-quickstart-test";
+		public static String empty_maven_project="empty-maven-project";
+		public static String other_empty_maven_project="empty-maven-project";
+		public static String folder_name_different_maven="folder-name-different-maven";
+		public static String hibernate_orm_resteasy="hibernate-orm-resteasy";
+		public static String hibernate_orm_resteasy_yaml="hibernate-orm-resteasy-yaml";
+		public static String kubernetes="kubernetes";
+		public static String microprofile_fault_tolerance="microprofile-fault-tolerance";
+		public static String microprofile_health_quickstart="microprofile-health-quickstart";
+		public static String microprofile_lra="microprofile-lra";
+		public static String microprofile_context_propagation="microprofile-context-propagation";
+		public static String microprofile_metrics="microprofile-metrics";
+		public static String microprofile_opentracing="microprofile-opentracing";
+		public static String microprofile_openapi="microprofile-openapi";
+		public static String microprofile_reactive_messaging="microprofile-reactive-messaging";
+		public static String microprofile_graphql="microprofile-graphql";
+		public static String microprofile_jwt_quickstart="microprofile-jwt-quickstart";
+		public static String quarkus_container_images="quarkus-container-images";

```

Comment:
I have an issue open to move test project names that are only used in quarkus ls over to the quarkus-ls repo #7

Would be nice to remove these from lsp4mp and include them as part of quarkus-ls.

ex. quarkus-container-images: https://github.com/redhat-developer/quarkus-ls/blob/7f2f111b725b9590dc89373293f375cb033ccd5f/quarkus.jdt.ext/com.redhat.microprofile.jdt.quarkus.test/src/main/java/com/redhat/microprofile/jdt/quarkus/containerimages/QuarkusContainerImagesTest.java#L35

Reply:
I moved `kubernetes` and `quarkus_container_images` as those were the ones I found whose references were only on the quarkus-ls side. If I've missed any, let me know. I also removed the corresponding projects in the lsp4mp tests since presumably they were using the project on the quarkus-ls side anyways. The tests seem to be passing.

I've also modified GradleProjectName in the same way but I haven't created a quarkus equivalent because I couldn't find anything that is used only on the quarkus-ls side.